packages/create-astro/src/messages.ts

```diff
@@ -77,7 +77,11 @@ let v: string;
 export const getVersion = () =>
 	new Promise<string>(async (resolve) => {
 		if (v) return resolve(v);
-		const registry = await getRegistry();
+		let registry = await getRegistry();
+		// if the registry ends with '/', trim it.
+		if (registry.lastIndexOf('/') === registry.length - 1) {
+			registry = registry.substring(0, registry.length - 1);
```

Comment:
nit: You could use [`trimEnd`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimEnd)

Reply:
Thank you very much for your help on this issue! I updated the logic to handle whitespace and trailing `/` in a centralized place.